src/index.html

```diff
@@ -100,24 +99,24 @@ <h6>Daily Minutes Active:
       <span class="detailed-activity min-active">0/all user avg</span>
     </h6>
     <h6>Daily Flight of Stairs:
-      <span class="detailed-activity min-active">0/all user avg</span>
+      <span class="detailed-activity flight-of-stairs">0/all user avg</span>
     </h6>
     <h6>Daily Mileage Walked: 
       <span class="detailed-activity mile-walk">0</span>
     </h6>
     <h5>Weekly Stats</h5>
     <h6>Step Count:
-      <div class="detailed-activity">
+      <div class="detailed-activity weekly-steps">
         <span>0</span>
       </div>
     </h6>
     <h6>Flights of Stairs Climbed:
-      <div class="detailed-activity">
+      <div class="detailed-activity weekly-stairs">
         <span>0</span>
       </div>
     </h6>
     <h6>Minutes Active:
-      <div class="detailed-activity">
+      <div class="detailed-activity weekly-mins-active">
```

Comment:
I noticed the lack of a class selector for DOM manipulation, too! 
Something I want to refactor is having a span in a div. I think we can just apply the classes to the span and delete the div and it won't have any DOM or styling repercussions.